config/configuration.json

```diff
@@ -189,6 +189,16 @@
       "tags": "highway"
     }
   },
+  "ConstructionCheck": {
+    "oldConstructionDays": 730.0,
```

Comment:
Just curious - whats the reasoning behind these values? Is 2 years a recommend length of time for checking these highways?

Reply:
Osmose uses 2 years as the limit for something to have been in construction when no date tags are available
https://github.com/osm-fr/osmose-backend/blob/master/plugins/Construction.py#L36

And the 6 month check_date was also from their check, though someone would have to update the check_date each time it gets flagged instead of marking as false positive:
https://github.com/osm-fr/osmose-backend/blob/master/plugins/Construction.py#L98